### DISCUSSION TITLE
Programmatically select a tab.

### DISCUSSION THREAD

**Discussion by corpulent** [tailwindlabs/headlessui#965] [Help]
Is there a workaround to programmatically select a tab?  I need to open a tab (if its not open) when a specific event happens.

**Comment by k-miyata** [comment]
It seems to be a duplicate of https://github.com/tailwindlabs/headlessui/discussions/742. A solution in https://github.com/tailwindlabs/headlessui/discussions/742#discussioncomment-1661403 may help you.
